### ISSUE TITLE
Play nicely with remux

### ISSUE THREAD

**Issue by raisjn** [LinusCDE/retris#1]
steps:

* install remux + retris
* launch remux by holding down center button
* launch retris from remux
* start a game, use swipes and see works as expected
* launch remux - see game pauses
* continue game, see game resumes
* notice that touch input is no longer working (swipes don't respond)

as a data point, this is not a problem for plato (which i think you also ported using libremarkable)

**Comment by LinusCDE** [comment]
I found the issue. Seems that libremarkable didn't understandably expect the epoll wait call to crash due to an Interruption:

```
RENDERING Remarkable
RENDERING reTris
LAUNCHING APP reTris
CHECKING PROCESS retris
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }', /root/.cargo/git/checkouts/libremarkable-6002586188ceddce/440a914/src/input/ev.rs:89:71
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }', /root/.cargo/git/checkouts/libremarkable-6002586188ceddce/440a914/src/input/ev.rs:89:71
retris IS ALREADY RUNNING, RESUMING
RENDERING reTris
```

I addressed this in PR https://github.com/canselcik/libremarkable/pull/44 and used my fix for the newest release (0.5.6-1) to fix this faster.

So this issue would affect every library that uses libremarkable until now. [plato](https://github.com/LinusCDE/plato/) isn't affected, since it doesn't use the ev implementation from libremarkable.

**Comment by LinusCDE** [comment]
Release 0.5.6-1 had still some trouble. Toltec did use a wrong commit to build libremarkable which didn't contain the fix. I updated the `Cargo.lock` in commit 3a5b2eb3 which resolves this (in release 0.5.7-1).
